app/views/tag/show.html.erb

```diff
@@ -95,6 +95,12 @@
     width: 40%;
     padding: 10px 10px !important;
   }
+  
```

Comment:
Hi, @NitinBhasneria -- thanks for this! I am concerned though that just changing the individual styles may not get us the smooth and consistent styling we want, and that the Froala example provides. It's been so thoroughly tested at different page widths, you know? And here, we'd be overriding a global style with a one-off fix. I'm really more inclined to aim to use standard Bootstrap and Froala CSS classes to solve this in a consistent and standard way. I hope you understand!

Reply:
Ok, so I will be trying froala design for this issue.
Thanks @jywarren